### DISCUSSION TITLE
预览时报错相同组件名

### DISCUSSION THREAD

**Discussion by stwflyfox** [opentiny/tiny-engine#659] [Q&A]
![QQ_1721117251199](https://github.com/user-attachments/assets/9a62a70b-1984-4340-a5fa-15e3bacf3b38)
其中一个是以前已经废弃的，在user_components表里现在也只有一个，另一个是从哪取到的？
![QQ_1721117785852](https://github.com/user-attachments/assets/0715a8a3-b97d-4722-8c21-958763291e92)

**Comment by chilingling** [comment]
新版出码的依赖声明语句跟 `componentsMap` 强相关，可以看看是不是 `componentsMap` 数据有误。

![image](https://github.com/user-attachments/assets/c768993b-6e44-4a8c-af9f-1cdc1616c689)

**Comment by stwflyfox** [comment]
已解决，区块中有历史版本的同名组件
